### PR TITLE
[Redesign]Autofocus search

### PR DESCRIPTION
**Pull request by ryuyu**
If no element has focus (or body has focus), typing will jump to search box.
As suggested in https://github.com/NuGet/NuGetGallery/issues/4458

@joelverhagen @microsoftsam

### CODE REVIEW

**Comment by microsoftsam** [comment]
Nice!

File: src/NuGetGallery/Scripts/gallery/common.js [hunk 1]
```diff
@@ -309,4 +309,12 @@ $(function () {
     $(document).on('click', '.dropdown-menu', function (e) {
         e.stopPropagation();
     });
+
+    $(document).on('keydown', function (e) {
+        var code = (e.keyCode || e.which);
+
```
Comment (joelverhagen) [inline src/NuGetGallery/Scripts/gallery/common.js]: Please comment about the `code != 9` part.

Comment (joelverhagen) [inline src/NuGetGallery/Scripts/gallery/common.js]: Code comment -- that's Tab.

Comment (joelverhagen) [inline src/NuGetGallery/Scripts/gallery/common.js]: Should we make this more generic? Perhaps the key should be alpha + numbers + symbols?

**Pull request merged**